modules/apps/portal-settings/portal-settings-web/src/main/java/com/liferay/portal/settings/web/internal/portal/settings/configuration/admin/display/DefaultPortletDecoratorPortalSettingsConfigurationScreenContributor.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.settings.web.internal.portal.settings.configuration.admin.display;
+
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.PrefsPropsUtil;
+import com.liferay.portal.kernel.util.PropsKeys;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.settings.configuration.admin.display.PortalSettingsConfigurationScreenContributor;
+import com.liferay.portal.util.PropsValues;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author João Victor Torres
+ */
+@Component(service = PortalSettingsConfigurationScreenContributor.class)
+public class DefaultPortletDecoratorPortalSettingsConfigurationScreenContributor
+	extends BaseEditCompanyPortalSettingsConfigurationScreenContributor {
+
+	@Override
+	public String getCategoryKey() {
+		return "look-and-feel";
+	}
+
+	@Override
+	public String getJspPath() {
+		return "/default_portlet_decorator.jsp";
+	}
+
+	@Override
+	public String getKey() {
+		return "default_portlet_decorator";
```

Comment:
@torresvictor100, the language keys are separated by `-` instead of `_`, can you please adjust it here and in the next commit?

Thanks